django/db/migrations/writer.py

```diff
@@ -350,6 +350,8 @@ def serialize(cls, value):
         # process.
         if isinstance(value, Promise):
             value = force_text(value)
+        elif isinstance(value, LazyObject):
+            value = value.__reduce__()[1][0]
```

Comment:
`LazyObject.deconstruct()` would have to use `MigrationWriter.serialize(self._wrapped)` to be effective since it can wrap any Python type.